src/lib/core/clock_lowres.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#pragma once
+
+/**
+ * Monotonic low resolution clock, based on interval timer.
+ * Not thread-safe!
+ */
+
+#if __cplusplus
+extern "C" {
+#endif
+
+#include <assert.h>
+#include <stdbool.h>
+
+/** Low resolution clock accumulator. */
+extern double clock_lowres_monotonic_clock_value;
+
+#ifndef NDEBUG
+
+/**
+ * Check that current thread is the thread
+ * that initialized this module.
+ */
+bool
+clock_lowres_thread_is_owner(void);
+
+#endif /* NDEBUG */
+
+/** Get resolution of clock_lowres clocks in double. */
+double
+clock_lowres_resolution(void);
+
+/**
+ * Blazingly fast low resolution monotonic time in seconds.
+ */
+static inline double
```

Comment:
AFAIC, you don't need `static` here, according to https://en.cppreference.com/w/c/language/inline:
> If a non-static function is declared inline, then it must be defined in the same translation unit. The inline definition that does not use extern is not externally visible and does not prevent other translation units from defining the same function. This makes the **inline** keyword an alternative to static for defining functions inside header files, which may be included in multiple translation units of the same program.